### ISSUE TITLE
Cannot get Signal UUID and password

### ISSUE THREAD

**Issue by swindlesmccoop** [laggykiller/sticker-convert#125]
I installed Signal Beta, but when I click the "get uuid and password" button, I get the following error on the terminal:
```
Signal Desktop is using chrome version 120
Warning: Cannot find binary file C:\Users\Swindles\Downloads\sticker-convert\bin\chromedriver.exe.exe
Exception in thread Thread-4 (cb_login_thread):
Traceback (most recent call last):
  File "C:\Users\Swindles\DOWNLO~1\STICKE~1\threading.py", line 1045, in _bootstrap_inner
  File "C:\Users\Swindles\DOWNLO~1\STICKE~1\threading.py", line 982, in run
  File "C:\Users\Swindles\DOWNLO~1\STICKE~1\sticker_convert\gui_components\windows\signal_get_auth_window.py", line 75, in cb_login_thread
  File "C:\Users\Swindles\DOWNLO~1\STICKE~1\sticker_convert\utils\auth\get_signal_auth.py", line 208, in get_cred
  File "C:\Users\Swindles\DOWNLO~1\STICKE~1\sticker_convert\utils\auth\get_signal_auth.py", line 323, in launch_signal_desktop
  File "C:\Users\Swindles\DOWNLO~1\STICKE~1\sticker_convert\utils\auth\get_signal_auth.py", line 109, in get_local_chromedriver
IndexError: list index out of range
```

**Comment by laggykiller** [comment]
Thank you for reporting, should be fixed in v2.6.5, please try!

**Comment by swindlesmccoop** [comment]
Fixed, thank you very much.

**Comment by vyeol** [comment]
Reviving this.
In app version 2.9.4, pressing the in-app "Generate" button to retrieve Signal uuid and password yields no reaction.
Signal Desktop (7.19.0) on macOS (Sonoma 14.6) is installed, linked to phone, and launched.

**Comment by laggykiller** [comment]
@vyeol Let's continue discussion in https://github.com/laggykiller/sticker-convert/issues/188

**Comment by laggykiller** [comment]
@vyeol the bug was fixed in v2.10.0, please try
